File: kyu8/even-or-odd.php
<?php /*
DESCRIPTION:
Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
*/
function even_or_odd(int $n): string {
	$a = $n%2;
  return $a === 0 ? 'Even' : 'Odd';
}

File: kyu8/even-or-odd.swift
/*
Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
*/
func evenOrOdd(_ number:Int) -> String {
    if number % 2 == 0 {
    return "Even"
  } else {
    return "Odd"
  
	}
}